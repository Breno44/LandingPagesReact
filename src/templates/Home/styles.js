import styled, { css } from "styled-components";

export const Wrapper = styled.div`
  display: flex;
  background-color: ${(props) => props.background};
`;
