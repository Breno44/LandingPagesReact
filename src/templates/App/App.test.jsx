import { screen } from "@testing-library/react";
import { Home } from ".";
import { renderTheme } from "../../styles/renderTheme";
import { theme } from "../../styles/theme";

test("renders learn react link", () => {
  renderTheme(<Home />);
  const headingContainer = screen.getByRole("heading", {
    name: "Hello",
  }).parentElement;

  expect(headingContainer).toMatchSnapshot();
});
