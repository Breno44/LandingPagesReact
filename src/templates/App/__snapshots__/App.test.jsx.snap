// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
.c0 {
  color: #0A1128;
  font-size: 4.0rem;
  text-transform: none;
}

<div
  class=""
>
  <h1
    class="c0"
  >
    Hello
  </h1>
</div>
`;
